exercises/simple-linked-list/example.py

```diff
@@ -0,0 +1,48 @@
+class Node(object):
+    def __init__(self, value):
+        self._value = value
+        self._next = None
+
+    def value(self):
+        return self._value
+
+    def next(self):
+        return self._next
```

Comment:
Personally, I would make `value` and `next` into properties/attributes rather than methods.

Reply:
The intent of making `value` and `next` methods was that they could be included as stubs in the solution template but their actual implementation (simple as they might be) could be left to the user.